blog-api/docker/dev/nginx/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM nginx:1.21.1-alpine
+
+RUN apk add --no-cache curl
```

Comment:
Why do we need curl in nginx image?

Reply:
For debug purposes in dev environment and for healthchecks in prod, but healthckecks are disabled in docker-compose.
When we migrate to swarm/k8s it should work.